script/DeploySafeNativeRecovery.s.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+import { SafeEmailRecoveryModule } from "src/modules/SafeEmailRecoveryModule.sol";
+
+contract DeploySafeNativeRecovery_Script is Script {
+    function run() public {
+        vm.startBroadcast(vm.envUint("PRIVATE_KEY"));
+        address manager = vm.envOr("VERIFIER", address(0));
+
+        if (manager == address(0)) {
+            manager = address(1);
```

Comment:
Is this address(1) as a placeholder to run this script?
Is it okay if we don't revert?

Reply:
Yeah this is just a placeholder, I need to update this PR to the latest changes from main